app/views/catalog/_zero_results.html.erb

```diff
@@ -1,4 +1,8 @@
-<h1><%= t('trln_argon.search.zero_results.title', local_institution_name: institution_long_name) %></h1>
+<% if request.env['PATH_INFO'] == '/trln' %>
```

Comment:
The view shouldn't care about the current request path. I would suggest a different approach.

Reply:
One option would be to add a new template to `app/views/trln/_zero_results.html.erb` that would apply to the TRLN view.

Reply:
I think I need more details here.

Reply:
What's wrong with this approach? Where do I check the trln view?  What would be a new structure of the files if I add a new template?

Reply:
Since the TrlnController is a sub class of CatalogController when rendering a trln view the application first looks in app/views/trln for a template. If it finds none there it falls back to one in app/views/catalog. So...if you add a template for no results to the app/views/trln path it will get rendered for trln views instead of the default one in the catalog view path. I'd be happy to do a quick zoom call if you want.